pkgs/tools/misc/lanzaboote/default.nix

```diff
@@ -0,0 +1,31 @@
+{ rustPlatform, stdenv, fetchFromGitHub, lib, breakpointHook }: rustPlatform.buildRustPackage {
+  pname = "lanzaboote-stub";
+  version = "0.3.0";
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = "lanzaboote";
+    rev = "83a357eb7c261c82e2fa6412216b3c4f8f0d1d18";
+    hash = "sha256-jWmTTUejSCa5qO0D/FJMbgXKdFEllRdvA+jG0KX+MfY=";
+  };
+  sourceRoot = "source/rust/stub";
+
+  cargoHash = "sha256-FlnheCgowYsEHcFMn6k8ESxDuggbO4tNdQlOjUIj7oE=";
+
+  # TODO: limit supported platforms to UEFI
```

Comment:
Missing the full meta and meta should be last.

Reply:
Yes, this is more a test package PoC to show the "value" of those changes rather than the real package yet